testing/scenario_app/ios/Scenarios/ScenariosUITests/GoldenImage.m

```diff
@@ -33,8 +33,14 @@ - (BOOL)compareGoldenToImage:(UIImage*)image {
     os_log_error(OS_LOG_DEFAULT, "GOLDEN DIFF FAILED: image does not exists.");
     return NO;
   }
-  CGImageRef imageRefA = [self.image CGImage];
-  CGImageRef imageRefB = [image CGImage];
+
+  // The home bar at the bottom of the screen can be different shades of gray
```

Comment:
I was hoping to use [`prefersHomeIndicatorAutoHidden`](https://developer.apple.com/documentation/uikit/uiviewcontroller/2887510-prefershomeindicatorautohidden) similar to 
https://github.com/flutter/engine/blob/9e49ffb5859f722890015c2172d6bca233f70cd1/testing/scenario_app/ios/Scenarios/Scenarios/AppDelegate.m#L17
But that makes it worse, the home indicator bar animates in and then animates out.

Reply:
This is ok to me.

Side node:
Is it possible to wait for the home indicator to be hidden with a `XCTestExpectation`?

Reply:
> Is it possible to wait for the home indicator to be hidden with a `XCTestExpectation`?

Good idea! I checked springboard and there's a "Home Grabber" element.